tests/test_wave_05.py

```diff
@@ -171,7 +171,6 @@ def test_get_new_rec_by_genre_returns_empty_list_when_friends_watched_lists_are_
     assert len(recommendations) == 0
 
```

Comment:
This file didn't need to be pushed. You can use commands like `git add <filename>` to ensure that you're only adding specific files to be tracked, committed, and pushed to Github.